guidelines/sc/22/focus-visible-enhanced.html

```diff
@@ -0,0 +1,15 @@
+<section class="sc">
+   
+    <h4>Focus Visible (Enhanced)</h4>
+    
+    <p class="conformance-level">AA</p>
+    
+    <p>When a <a>User Interface Component</a> receives keyboard <a>focus</a>, all of the following are true:</p>
```

Comment:
This is a change, but how about "When a User Interface Component displays a visible keyboard focus, all ..."

This SC currently omits the exception that is part of 2.4.7, and I think that we need it, so my change makes the enhanced SC build on the 2.4.7 SC.

Otherwise we will have people saying, "yes, 2.4.7 allows a mode where the focus is visible, but the new SC prohibits that practice".

Reply:
Do you mean "2.4.7 allows a mode where the focus is **in**visible, but the new SC prohibits that practice"?

A previous verion copied 2.4.7 more closely, using 'keyboard operable user interface".

I've not come across a site that has separate modes of operation, so I don't feel strongly, but don't mind either way.

Reply:
Yes, I meant that 2.4.7 allows a site to have a mode without a visible focus, so long as there is a mode to enable a visible focus (and an accessible way to get to the controls for that).